doc/examples/segmentation/plot_regionprops_table.py

```diff
@@ -0,0 +1,106 @@
+"""
+===================================================
+Explore and visualize region properties with pandas
+===================================================
+
+This toy example shows how to compute the size of every labelled region in a
+series of 10 images. We use 2D images and then 3D images. The blob-like
+regions are generated synthetically. As the volume fraction (i.e., number of
+pixels or voxels covered by the blobs) increases, the number of blobs
+(regions) decreases, and the size (area or volume) of a single region can get
+larger and larger. The area (size) values are available in a pandas-compatible
+format, which makes for convenient data analysis and visualization.
+
+Beyond area, many other region properties are available.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from skimage import data, measure
+
+
+fractions = np.linspace(0.05, 0.5, 10)
+
+#####################################################################
+# 2D images
+# =========
+
+images = [data.binary_blobs(volume_fraction=f) for f in fractions]
+
+labeled_images = [measure.label(image) for image in images]
+
+properties = ['label', 'area']
+
+tables = [measure.regionprops_table(image, properties=properties)
+          for image in labeled_images]
+tables = [pd.DataFrame(table) for table in tables]
+
+for fraction, table in zip(fractions, tables):
+    table['volume fraction'] = fraction
+
+areas = pd.concat(tables, axis=0)
+
+# Divide the figure into a 2x1 grid
+fig, (im, ax) = plt.subplots(2, 1)
+# Turn off axes for top subplot
+im.axis('off')
+# Plot area vs volume fraction
+areas.plot(x='volume fraction', y='area', kind='scatter', ax=ax)
+# Show image with lowest volume fraction
+ax1 = fig.add_subplot(221)
```

Comment:
It is a bit confusing that you are using two different ways of allocating the subplots: `fig, (im, ax) = plt.subplots(2, 1)` and `ax1 = fig.add_subplot(221)`. `plt.subplots` could be used solely, see https://github.com/scikit-image/scikit-image/blob/master/doc/examples/numpy_operations/plot_view_as_blocks.py#L46-L52 for the example.

Reply:
I didn't know of the `ravel` method, thanks. If I start with a 2x2 grid of subplots, I don't know how to 'merge' the bottom subplots to get this rendering:
![add_subplot](https://user-images.githubusercontent.com/2227806/71780388-6ac9ea80-2fc2-11ea-9951-793cefa1229d.png)
Using
```py
fig, axes = plt.subplots(2, 2)
ax = axes.ravel()
# Plot area vs volume fraction
areas.plot(x='volume fraction', y='area', kind='scatter', ax=ax[2])
# Show image with lowest volume fraction
ax[0].axis('off')
ax[0] = fig.add_subplot(221)
ax[0].imshow(images[0], cmap='gray_r')
ax[0].set_axis_off()
ax[0].set_title(f'fraction {fractions[0]}')
# Show image with highest volume fraction
ax[1].axis('off')
ax[1] = fig.add_subplot(222)
ax[1].imshow(images[-1], cmap='gray_r')
ax[1].set_axis_off()
ax[1].set_title(f'fraction {fractions[-1]}')
plt.show()
```
I get
![ravel](https://user-images.githubusercontent.com/2227806/71780431-86cd8c00-2fc2-11ea-9543-08c2cb5d28bb.png)
... as expected. Morally, I would like to express (in quotes) `areas.plot(x='volume fraction', y='area', kind='scatter', "ax=ax[2] + ax[3]")`: Do you know how to achieve this?

Reply:
@mkcor I think the canonical way is to use gridspec: https://matplotlib.org/3.1.1/tutorials/intermediate/gridspec.html

I don't know of a way to use `plt.subplots` and then join two plots. I don't think that exists. Having said that, I dislike the `add_subplot(221)` syntax and prefer to use `add_subplot(2, 2, 1)` if necessary. ;)

Reply:
I think GridSpec is the way to go. It is unlikely that we can merge subplots after `.subplots` call 🙂 .

Reply:
Here is an amazing recent post in matplotlib blog, btw - https://matplotlib.org/matplotblog/posts/an-inquiry-into-matplotlib-figures/ .

Reply:
Thank you, @soupault and @jni! Done in e6ed72a38ed22abf27cbe3e6dbee7fed3be9741b. I confess that I followed https://jakevdp.github.io/PythonDataScienceHandbook/04.08-multiple-subplots.html, not just because I :heart: @jakevdp, but also to save an extra import.